doc/00-onboarding-guide.md

```diff
@@ -1,30 +1,31 @@
 
-Send mail to amecodegen@microsoft.com when your swagger is ready, then we will
+## Step 1. Send request to AME team in Github
+AME team use github issue as a centralized entry point to trigger the process for new RP onboarding or add feature in existing RP. Please go to [Azure-Cli](https://github.com/Azure/azure-cli) to create a new github issue and following the onboarding template first. 
+With the selection result in template, if you select Azure cli Code-gen as the approach, you will be re-direct back to this document later.
+> ** we are improving the process here with Azure CLI team soon 
 
+## Step 2. Quick Validation
+If it is your first time to use Azure Cli code-gen, please discuss with the contact person. He/She will: 
 1. Do a quick validation on your swagger and prepare initial readme files for you
-2. Have one person assigned for your service to follow up with your for any questions and support needed
-
-## Step 2. Generate code and give it a try
+2. follow up with you for any questions and support needed during the service onboarding
 
+## Step 3. Generate code and give it a try
 1. There are 3 ways to generate code now, you can pick the one you prefer. Details can be found at [here](how-to-generate.md)
 2. Try the generated command to see whether it meets your need. Further customization is supported if needed. Detail can be found at [FAQ](faq.md).
 
-## Step 3. Run Test
-1. Test will also be generated based on the examples in swagger. Run these test and make sure all of them can pass. (Refer to [here](how-to-generate.md#through-our-pre-prepared-docker) for how to run test). Customization is also supported for test if needed. Detail can be found at [here](04-scenario-test-configuration.md)
+## Step 4. Run Test
+1. Test will also be generated based on the examples in swagger. Run these tests and make sure all of them can pass. (Refer to [here](how-to-generate.md#through-our-pre-prepared-docker) for how to run test). Customization is also supported for test if needed. Detail can be found at [here](04-scenario-test-configuration.md)
 2. Add more test if the generated test is not enough. Detail can be found at [here](https://github.com/Azure/azure-cli/blob/dev/doc/authoring_tests.md)
 
-## Step 4. Onboarding to azure-cli
-1. Follow [azure-cli guidance](https://github.com/Azure/azure-cli/blob/dev/doc/onboarding_guide.md) to onboard to azure-cli
-> ** we are improving the process here with Azure CLI team now
-
-## Step Next. CLI is ready when swagger is ready
-Autorest.az has been integrated into swagger PR. It means you can start reviewing your CLI code/command immediately when your swagger PR is filed for your new features/changes, so your new features/changes can have Azure CLI support very soon. 
-
-Hopefully, **your CLI is ready to merge/release when your swagger is ready to merge. :shipit: :+1:** 
+## Step 5. Run Linter and Style
```

Comment:
>Run Linter and Style [](start = 11, length = 20)

let's merge this into Step4 to make things look simpler

Reply:
ok

---
In reply to: [449504555](https://github.com/Azure/autorest.az/pull/461#discussion_r449504555) [](ancestors = 449504555)